modules/common/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@erento/nestjs-common",
+  "version": "1.0.0",
+  "description": "NestJS module with some shared utilities",
+  "main": "dist/public_api.js",
+  "types": "dist/public_api.d.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "lint": "tslint -c tslint.json --fix -e \"**/node_modules/**\" -p ./tsconfig.json",
```

Comment:
dont we usually have 2 scripts, one just for lint and lint:fix?

Reply:
we don't have that in nestjs-modules because those projects are usually small and therefore is not a big problem to run full fix always

we would need to change all other modules